src/Umbraco.Web.UI.Client/src/views/propertyeditors/grid/grid.prevalues.html

```diff
@@ -149,7 +148,7 @@ <h4><localize key="grid_rowConfigurations" /></h4>
 
             <ul class="unstyled list-icons">
                 <li>
-                    <i class="icon icon-settings-alt-2 turquoise"></i>
+                    <i class="icon icon-settings-alt-2 turquoise" aria-hidden="true"></i>
                     <a href="#" ng-click="editStyles()" prevent-default>
```

Comment:
And finally this should be a `<button>` too :-) - Also it's a good idea to wrap a fallback text in the `<localize>` element since it will currently not hold a fallback value if  general_edit is missing in a language file. So it's better to have `<localize key="general_edit">Edit</localize>` for instance ensuring that if the key is missing in Danish then at least "Edit" will be rendered instead of nothing